ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -174,7 +174,7 @@ class TransactionDetailsScreen extends React.Component<Props> {
           transactionWallet ? (
             <RotatedCards cardType="Preview" wallets={[transactionWallet]} />
           ) : (
-            undefined
+            <RotatedCards cardType="Preview" />
```

Comment:
I think the ternary operator now is not needed anymore, you can render always <RotatedCards cardType="Preview" wallets={[transactionWallet]} />

Reply:
But it means I need to assign to wallets a possible undefined object (so changing wallet types on the RotatedCards component and implements a little bit of logic inside it to check it). Do you prefer anyway this approach?